src/app/workflow-center/custom-task/custom-task.component.html

```diff
@@ -0,0 +1,130 @@
+        </clr-dg-column>
+        <clr-dg-column [clrDgSortBy]="implementsTaskComparator">
+            <ng-container *clrDgHideableColumn="{hidden: false}">ImplementsTask</ng-container>
+            <clr-dg-string-filter [clrDgStringFilter]="implementsTaskFilter">
+            </clr-dg-string-filter>
+          </clr-dg-column>
+        <clr-dg-column>
+          <ng-container *clrDgHideableColumn="{hidden: false}">Options</ng-container>
+          <clr-dg-string-filter [clrDgStringFilter]="optionsFilter">
+          </clr-dg-string-filter>
+        </clr-dg-column>
+        <clr-dg-column >
+          <ng-container *clrDgHideableColumn="{hidden: false}">Properties</ng-container>
+          <clr-dg-string-filter [clrDgStringFilter]="propertiesFilter">
+          </clr-dg-string-filter>
+        </clr-dg-column>
+  
+        <clr-dg-row *clrDgItems="let task of tasksStore" [clrDgItem]="task">
+          <clr-dg-action-overflow>
+            <button class="action-item" (click)="onUpdate(task)">Update</button>
+            <button class="action-item" (click)="onDelete(task)">Delete</button>
+            <button class="action-item" (click)="onGetDetails(task)">Detail</button>
+          </clr-dg-action-overflow>
+          <clr-dg-cell>{{task.friendlyName}}</clr-dg-cell>
+          <clr-dg-cell >{{task.injectableName}}</clr-dg-cell>
+          <clr-dg-cell >{{task.implementsTask}}</clr-dg-cell>
+          <clr-dg-cell (click)="getChild('options', task)">
+            <u *ngIf="task.options  && (task.options | json) != '{}'" class="grid-item">Options</u>
+          </clr-dg-cell>
+          <clr-dg-cell (click)="getChild('properties', task)">
+            <u *ngIf="task.properties && (task.properties | json) != '{}'" class="grid-item">Properties</u>
+          </clr-dg-cell>
+        </clr-dg-row>
+  
+        <clr-dg-footer></clr-dg-footer>
+      </clr-datagrid>
+  
+    <div class="selected">
+      Selected Tasks:
+      <span *ngFor="let task of selectedTasks; let isLast = last">{{task? task.injectableName : null}}{{isLast ? '' : ', '}}</span>
+    </div>
+  </div>
+
+  <details-modal *ngIf="modalTypes.detailActions.indexOf(action) !== -1"
+    [(isPop)]="isShowModal" [data]="rawData"
+    [dataType]="action" [title]="selectedTask.injectableName">
+  </details-modal>
+
+  <clr-modal *ngIf="action === 'Delete'" [(clrModalOpen)]="isShowModal" [clrModalSize]="'xl'">
+    <h3 class="modal-title">Are you sure you want to delete below tasks</h3>
+    <div class="modal-body">
+      <span *ngFor="let task of selectedTasks; let isLast = last">
+        {{task? task.injectableName : null}}{{isLast ? '' : ', '}}
+      </span>
+    </div>
+    <div class="modal-footer">
+      <button class="btn btn-primary" type="button" (click)="onDeleteSubmit()">Confirm</button>
+    </div>
+  </clr-modal>
+
+
+  <clr-modal *ngIf="modalTypes.formActions.indexOf(action) !== -1"
+  [(clrModalOpen)]="isShowModal"
+  [clrModalSize]="'lg'">
```

Comment:
this could be `clrModalSize="lg"`

Reply:
all this mistakes comes from rackHD project. Maybe we should do a refactor to clean all of them (they are quite a lot). Should be open an issue?